### DISCUSSION TITLE
Documentation correction - object detectors

### DISCUSSION THREAD

**Discussion by halabut69** [blakeblackshear/frigate#15872] [General]
In the Object Detectors documentation under OpenVino there is a Google Collab workbook for exposing the pre-trained YOLO-NAS model as .onnx. This workbook fails to download the model as the hardcoded URL in the super-graduents library is incorrect. This should be corrected in the next release.
There is a branch with the error fixed here: https://github.com/hannadiamond/super-gradients
Adding the following line to the Google Collab workbook fixes the issue. 
`! pip install -U git+https://github.com/hannadiamond/super-gradients@2235adeacc876dbab442096085fc404e79ce19a4`
